### PR TITLE
-Copy todo-list

### PR DESCRIPTION
**Pull request by Zibilyonik**

### CODE REVIEW

**Review by moise10r — CHANGES_REQUESTED** [review]
#  Changes required :recycle:  

## Hey @Zibilyonik ,

125811485-7f6e14c7-10a8-4701-b6c1-fcb37e47538d

You have done a great job so far :clap: :clap: 

 Good points: :+1: 
- Your Readme is professional :heavy_check_mark: 
- Best used of ES6 sytanxes. :heavy_check_mark: 
- You used proper GitHub Flow :heavy_check_mark: 
- Your linter checks are all passing :heavy_check_mark: 


However, there is something that you still need to improve in order to fulfil all requirements!


SUGGESTIONS 
- It's a good practice to commit every time you include the changes in your project.
- Your PR message descriptive should be more descriptive :wink: 

CHANGE REQUIRED
Have a look at the comment below to see what was required

Happy coding computer

Best,
Moise

File: src/adjustItems.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  arr[index - 1].description = obj.innerText;
+  localStorage.clear();
+  localStorage.setItem('todoArray', JSON.stringify(arr));
+}
+
+function deleteMarked(arr) {
+  const bonk = arr.filter((item) => item.completed === false);
+  arr = [...bonk];
+  localStorage.clear();
+  localStorage.setItem('todoArray', JSON.stringify(arr));
```
Comment (moise10r) [inline src/adjustItems.js]: It could be better if you could create a function on setting and getting items on the local storage.

**Review by moise10r — CHANGES_REQUESTED** [review]

File: src/adjustItems.js (hunk 1 above)
Comment (Zibilyonik) [reply src/adjustItems.js]: storageSetter function is now added to address this issue. Thank you for bringing it up 👍

Comment (moise10r) [reply src/adjustItems.js]: That is really nice, I appreciate the effort that you have done so far. Keep it :100: :green_apple:

**Pull request merged**